src/modules/85-cv/components/LogsAnalysis/components/LogAnalysisRow/LogAnalysisRow.tsx

```diff
@@ -5,138 +5,193 @@
-      <Container
-        className={cx(css.logRowText, css.dataColumn, css.openModalColumn, css.count)}
-        onClick={onShowRiskEditModalCallback}
-        padding={{ left: 'medium' }}
-      >
-        <p className={css.count}>{rowData.count}</p>
-      </Container>
-      <Container className={cx(css.lineChartContainer, css.dataColumn, css.messageFrequency)}>
+      {/* <Container className={cx(css.lineChartContainer)}>
         <HighchartsReact highchart={Highcharts} options={chartOptions} />
-      </Container>
-      {displayRiskEditModal ? (
-        <LogAnalysisRiskAndJiraModal
-          onHide={onHideRiskEditModalCallback}
-          trendData={chartOptions}
-          count={rowData.count || 0}
-          activityType={rowData.clusterType}
-          logMessage={rowData.message || ''}
-          feedback={feedbackGiven}
-        />
-      ) : null}
+      </Container> */}
+      <span />
+      <Layout.Horizontal margin={{ top: 'xsmall' }} style={{ alignItems: 'center', justifyContent: 'flex-end' }}>
+        <Icon name="description" size={24} />
+      </Layout.Horizontal>
     </Container>
   )
 }
 
 export function LogAnalysisRow(props: LogAnalysisRowProps): JSX.Element {
-  const { data = [], isErrorTracking } = props
+  const { data = [], isErrorTracking, logResourceData, selectedLog, activityId, resetSelectedLog, goToPage } = props
   const [dataToCompare, setDataToCompare] = useState<CompareLogEventsInfo[]>([])
 
+  const [riskEditModalData, setRiskEditModalData] = useState<{
+    showDrawer: boolean
+    selectedRowData: LogAnalysisRowData | null
+  }>({
+    showDrawer: false,
+    selectedRowData: null
+  })
+
+  const { accountId } = useParams<AccountPathProps>()
+
+  const {
+    data: logsData,
+    loading: logsLoading,
+    error: logsError,
+    refetch: fetchLogAnalysis
+  } = useGetVerifyStepDeploymentLogAnalysisRadarChartResult({
+    verifyStepExecutionId: activityId as string,
+    queryParams: {
+      accountId
+    },
+    queryParamStringifyOptions: {
+      arrayFormat: 'repeat'
+    },
+    lazy: true
+  })
+
+  useEffect(() => {
+    let drawerData: LogAnalysisRowData = {} as LogAnalysisRowData
+
+    if (!logsLoading && logsData) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      const dataToDrawer = logsData.resource?.logAnalysisRadarCharts?.content[0]
+
+      drawerData = getSingleLogData(dataToDrawer as LogAnalysisRadarChartListDTO)
+
+      setRiskEditModalData({
+        showDrawer: true,
+        selectedRowData: drawerData
+      })
+    }
+  }, [logsData])
+
+  const retryLogsCall = useCallback(() => {
+    fetchLogAnalysis({
+      queryParams: {
+        accountId,
+        clusterId: selectedLog as string
+      }
+    })
+  }, [fetchLogAnalysis, accountId, selectedLog])
+
+  const onDrawerHide = useCallback(() => {
+    setRiskEditModalData({
+      showDrawer: false,
+      selectedRowData: null
+    })
+    resetSelectedLog?.()
+  }, [])
```

Comment:
Codacy has a fix for the issue: [React Hook useCallback has a missing dependency: 'resetSelectedLog'. Either include it or remove the dependency array.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232318)
```suggestion
  }, [resetSelectedLog])
```